notebooks/tutorials/data-engineer/02-deployment-types.ipynb

```diff
@@ -8,20 +8,193 @@
     "# Deployment Types"
```

Comment:
<p>This isnt a single container setup, should be something like</p><p><code></code><code></p><p>docker run -it -e DEFAULT_ROOT_PASSWORD=secret -e PORT=8080 -p 8080:8080 openmined/grid-enclave:0.8.1</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7861/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Updated!